src/applications/proxy-rewrite/redirects/crossDomainRedirects.json

```diff
@@ -1098,5 +1098,138 @@
     "domain": "www.benefits.va.gov",
     "src": "/gibill/vrrap.asp",
     "dest": "/education/other-va-education-benefits/veteran-rapid-retraining-assistance/"
+  },
+  {
+    "domain": "www.altoona.va.gov",
+    "src": "/",
+    "dest": "/altoona-health-care/",
+    "catchAll": true
+  },
+  {
+    "domain": "www.altoona.va.gov",
+    "src": "/locations/directions.asp",
+    "dest": "/altoona-health-care/locations/james-e-van-zandt-veterans-administration-medical-center/"
```

Comment:
Let's remove this 1:1 redirect, we do not need to redirect the legacy directions page.